### ISSUE TITLE
Micro G patches

### ISSUE THREAD

**Issue by Adity1243** [YT-Advanced/YT-Advanced#34]
Bring back micro g patches that was available before, like hide icon in launcher,change micro g icon and remove asking permission every time when open youtube for other gmail accounts in notification panel.

**Comment by Wojtaz0w** [comment]
Are you too stupid to just disable notifications from MicroG? Btw change your theme, its so damn ugly

**Comment by Adity1243** [comment]
Yes i am using it but bring back the micro g patches like change icon,and hide icon from launcher.
